README.md

```diff
@@ -17,16 +17,28 @@ If running on openshift simply run `make` which will build and push the image to
 
 ## **Usage**
 ### **Usage on openshift platform**
-Once we have OLS deployed and running on openshift cluster, in order to trigger the load test, deploy [assets/ols-load-generator.yaml](https://github.com/vishnuchalla/ols-load-generator/blob/perf-testing/assets/ols-load-generator.yaml) replaced with your corresponding envs values onto the cluster.
+Once we have OLS deployed and running on openshift cluster, in order to trigger the load test follow the below steps.
 
-### **Envs**
+
+#### **Create a secret with your cluster kubeconfig**
+```
+oc create secret generic kubeconfig-secret --from-file=kubeconfig=<YOUR-KUBECONFIG-PATH> -n ols-load-generator
+```
+
+#### **Deploy [config/ols-load-generator.yaml](https://github.com/openshift/ols-load-generator/blob/main/config/ols-load-generator.yaml) replaced with your corresponding envs values onto the cluster**
+
+#### **Envs**
 * `OLS_TEST_HOST` - String indicating OLS endpoint to perform load testing.
-* `OLS_TEST_RUNID`(Optional) - String specifying an unique ID. Will be helpful while comparing two runs or while looking at a specific run results.
+* `OLS_TEST_UUID`(Optional) - String specifying an unique ID. Will be helpful while comparing two runs or while looking at a specific run results.
 * `OLS_TEST_AUTH_TOKEN` - OLS auth token string.
 * `OLS_TEST_HIT_SIZE` - Indicates the total amount of requests to be sent as a part of the load testing.
 * `OLS_TEST_RPS` - Requests per second to be sent to the OLS endpoint in parallel.
+* `OLS_TEST_METRIC_STEP` - Step size for the cluster prometheus metrics to be captured.
+* `OLS_TEST_PROFILES` - List of metric profiles that contain queries to be executed on prometheus.
+* `OLS_TEST_ES_HOST`(Optional) - Elastic search host url. If not specified metrics will be indexed locally.
+* `OLS_TEST_ES_INDEX`(Optional) - Elastic search index name to store the data. If not specified metrics will be indexed locally.
 
-### **Example Usage**
+#### **Example Usage**
```

Comment:
The below line should be ~/config/ols-load-generator.yaml ?

Reply:
Updated as suggested.